cmd/inspect/inspect_policy_data.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// Define the `ec inspect policy-data` command
+package inspect
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"path/filepath"
+	"strings"
+
+	hd "github.com/MakeNowJust/heredoc"
+	"github.com/ghodss/yaml"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+
+	"github.com/hacbs-contract/ec-cli/internal/policy/source"
+	"github.com/hacbs-contract/ec-cli/internal/utils"
+)
+
+func inspectPolicyDataCmd() *cobra.Command {
+	var (
+		sourceUrls   []string
+		destDir      string
+		outputFormat string
+	)
+
+	validFormats := []string{"json", "yaml"}
+
+	cmd := &cobra.Command{
+		Use:   "policy-data --source <source-url>",
+		Short: "Read policy data from source urls and displays the data",
+
+		Long: hd.Doc(`
+			Read policy data from source urls and displays the data.
+
+			This fetches policy sources similar to the 'ec fetch policy' command, but once
+			the policy is fetched it reads json and yaml files inside the policy source and
+			displays the data.
+
+			Note that this command is not typically required to verify the Enterprise
+			Contract. It has been made available for troubleshooting and debugging purposes.
+		`),
+
+		Example: hd.Doc(`
+			Print data from a given source url:
+
+			  ec inspect policy-data --source quay.io/hacbs-contract/ec-policy-data
+		`),
+
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !slices.Contains(validFormats, outputFormat) {
+				return fmt.Errorf("invalid value for --output '%s'. accepted values: %s", outputFormat, strings.Join(validFormats, ", "))
+			}
+
+			if destDir == "" {
+				workDir, err := utils.CreateWorkDir(afero.NewOsFs())
```

Comment:
A `defer` to cleanup the workdir could be nice. This should take the fs from context via `util.FS`

Reply:
Done.